static/js/components/window.vue

```diff
@@ -68,15 +68,15 @@
               v-model="period" 
               label="Period" 
               field="period" 
-              desc="Fractional hours"
+              desc="Decimal hours"
               :errors="errors.period"               
               @input="update"
             />
             <customfield v-show="cadence === 'simple'"
               v-model="jitter" 
               label="Jitter" 
               field="jitter" 
-              desc="Acceptable deviation (before or after) from strict period."
+              desc="Acceptable deviation in decimal hours (before or after) from strict period."
```

Comment:
I would phrase this "Acceptable deviation from strict period (before or after) in decimal hours.

Reply:
Done